### DISCUSSION TITLE
Set limit on file size for Attachment field using BudibaseDB

### DISCUSSION THREAD

**Discussion by eyduh** [Budibase/budibase#8997] [Help] [answered]
Hi,

I'm looking for a way to limit the file size for uploads in an attachment field. I have had it that users put very large files and click to create rows multiple times as the interface is stuck trying to upload very large files.

How would I go about doing this?

Cheers,
eyduh

**Comment by andz-bb** [comment, marked as answer]
Hey @eyduh, it's not currently possible but an issue has been raised for it here: #8936 

You can comment your interest there and hopefully it will be picked up soon!
